### DISCUSSION TITLE
Asi bac 855 costum firmware

### DISCUSSION THREAD

**Discussion by Bafanger** [danielnilsson9/bbs-fw#115] [Ideas]
Hi iv seen a lot of not user friendly controllers and asi is on top. I was wondering if its possible to get a custom firmware on it ?

**Comment by danielnilsson9** [comment]
Probably possible unless it requires encrypted firmware or is write protected, but I doubt it.
This firmware could probably be ported if anyone is interested in doing that and maintaining it, but it's a lot of work
